lib/wab/impl/model.rb

```diff
@@ -21,7 +22,7 @@ def initialize(dir)
         @cnt = 0
         @map = {}
         @lock = Thread::Mutex.new
-        Dir.mkdir(@dir) unless @dir.nil? || Dir.exist?(@dir)
+        FileUtils.mkdir_p(@dir) unless @dir.nil? || Dir.exist?(@dir)
```

Comment:
Much better.

Reply:
For some reason, `Dir.mkdir` chokes when I try to run the tests locally.. so there's every chance Travis won't run properly without this change..

Reply:
The key there is mkdir will not make the parent directories if they are not there and mkdir_p will.

Reply:
I agree.. but it works fine on the `develolp` branch..

Reply:
Maybe the parent directory was already there.